### ISSUE TITLE
Undefined column "created_at" when trying to use latest() on a table without timestamps

### ISSUE THREAD

**Issue by vicenterusso** [TiagoSilvaPereira/vemto-issues#265]
With a simple table ([schema](https://i.imgur.com/IcwDlgL.png)), I can't access the index page (my case `/bancos`). I got the following error

```sql
SQLSTATE[42703]: Undefined column: 7 ERROR: column "created_at" does not exist LINE 1: ..."::text like $2 
or "nome"::text like $3) order by "created_a... ^ (SQL: select * from "bancos" where ("id"::text like %% or
"codigo"::text like %% or "nome"::text like %%) order by "created_at" desc limit 5 offset 0)
```

Below is the generated code. If I change the `latest()` function with `latest('id')` the problem is gone

```php
class BancoController extends Controller
{
    /**
     * @param \Illuminate\Http\Request $request
     * @return \Illuminate\Http\Response
     */
    public function index(Request $request)
    {
        $this->authorize('view-any', Banco::class);

        $search = $request->get('search', '');

        $bancos = Banco::search($search)
            ->latest()
            //->latest('id) -> this works
            ->paginate(5)
            ->withQueryString();

        return view('app.bancos.index', compact('bancos', 'search'));
    }
}
```

**Comment by vicenterusso** [comment]
Tweaked a little bit and solved changing the `latest()` line with:

```php
->latest(<% if(!this.model.timestamps) { %>'<$ this.model.getPkName() $>'<% } %>)
```

Just don't know if there's a better way

**Comment by TiagoSilvaPereira** [comment]
@vicenterusso you did the correct fix... I'll bring something similar on the next version

**Comment by TiagoSilvaPereira** [comment]
Fixed on 1.0.5
